### PR TITLE
Add author tags for relevant files

### PR DESCRIPTION
**Pull request by charmainehly**
Some author tags created to fix the authorship for code repo!

Tagging you guys as I added your names to some parts of the files that originally credited me due to the refactoring..

### CODE REVIEW

**Review by domlimm — APPROVED** [review]
Wait! We need to fix authorship?! 😢  I thought reposense covers that. But thanks @charmainehly! Approved.

Either @lzan98 or @tiewweijian can merge.

**Comment by charmainehly** [comment]
> Wait! We need to fix authorship?! 😢 I thought reposense covers that. But thanks @charmainehly! Approved.
> 
> Either @lzan98 or @tiewweijian can merge.

We only need to fix if there are any issues! I think due to the refactoring i was credited for some files that I really didnt touch aside from the `persons` to `candidates` change, hence I decided to put in the empty author tag for those. But some of you guys edited the files, so I wanted to be sure to tag yall!!

Maybe hang on for all of you guys to check before the merge just to be sure I got the tagging done right! :)

File: src/main/java/seedu/address/model/ReadOnlyInterviewSchedule.java
Comment (lzan98) [inline src/main/java/seedu/address/model/ReadOnlyInterviewSchedule.java:13]: Thanks for making the change in the JavaDocs😄

**Review by lzan98 — APPROVED** [review]
LGTM 👍 Will wait for @tiewweijian to check and merge

File: src/main/java/seedu/address/model/ReadOnlyInterviewSchedule.java
Comment (charmainehly) [reply src/main/java/seedu/address/model/ReadOnlyInterviewSchedule.java:13]: no probs just spotted it while going thru :)

**Comment by domlimm** [comment]
> > Wait! We need to fix authorship?! 😢 I thought reposense covers that. But thanks @charmainehly! Approved.
> > Either @lzan98 or @tiewweijian can merge.
> 
> We only need to fix if there are any issues! I think due to the refactoring i was credited for some files that I really didnt touch aside from the `persons` to `candidates` change, hence I decided to put in the empty author tag for those. But some of you guys edited the files, so I wanted to be sure to tag yall!!
> 
> Maybe hang on for all of you guys to check before the merge just to be sure I got the tagging done right! :)

Thanks for clarifying @charmainehly! But there's too many files already :') gosh

**Pull request merged**
